docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+
+![zgw-in-gegevenslandschap.png](./bestanden/zgw-in-gegevenslandschap.png?raw=true)
+*Figuur 2 - ZGW in een Gegevenslandschap*
+
+* De domein/objectgegevens gegevens zijn uit de Zaakafhandelcomponent verdwenen. Procesdefinities en procesgegevens blijven daar echter achter .
+* Uitgangspunt in het gegevenslandschap is: alle gegevens worden op één plaats opgeslagen. En alle soortgelijke gegevens worden bij elkaar opgeslagen. Er komt dus een registratie met besluiten, een registratie met aanvragen en meldingen, een registratie met klanten, een registratie medewerkers. Etc.
+* De “Zaak” is een belangrijk verbindend element tussen de objecten in de registraties. Getekend: een zaak aggregeert al deze andere objecten. Zaak fungeert zo als “sleutel” in een grote “verwijsindex” naar objecten in andere registraties.  Opmerking AK En wat moet ik me daarbij voorstellen, wat is het nut daarvan? Welke op hun beurt weer kunnen relateren naar concepten in andere registraties, bv. de BRP of het HR. Opmerking AK Er zijn ook andere verbindende objecten zoals de klant (of een NP of NNP) die een mail gestuurd heeft (document), houder is van een vergunning, een handhavingsverzoek heeft ingediend (aanvraag/melding) en een paspoort heeft gekregen. Dergelijke relaties zijn in de figuur niet gevisualiseerd. Terecht, de focus is zaakgericht werken. Maar wel even vermelden.
+* Documenten die bij een zaak horen kunnen dan in de DRC worden opgehaald mbv het zaaknummer. Idem voor besluit(en) per zaak en betrokkenen.
+* Dit komt sterk overeen met het concept Linked-data
+Opmerking AK Wat is de overeenkomst dan? M.i. is het simpelweg een objecten- of informatiemodel met objecten en relaties. LinkedData is dan een manier om gebruik te kunnen maken van de in die modellen onderscheiden relaties. De Zakenregistratiecomponent is dus niet meer één grote database met daarin alle gegevens over een zaak. Maar meer een kernregistratie van zaken met veel links (URL’s) naar objecten in andere registraties (dan wel verwijzen andere objecten naar zaken).
+Opmerking AK Essentie hiervan is dat objecten die nu in het RGBZ en daarmee in de ZRC voorkomen maar ook relevantie hebben in een andere context of domein, als aparte registratie neergezet worden om recht te doen aan de betekenis als zelfstandige objecttype.
+* Dit kunnen zijn (niet uitputtend):
+    * Kernregistratie Klanten en Contacten (voor initiators,  belanghebbenden, gemachtigden e.d.)
+    * Kernregistratie medewerkers (o.a. voor behandelaars)
+    * Kernregistratie besluiten (besluiten)
+* Documentregistratiecomponent (documenten cq. informatieobjecten)
+    * Zaaktyperegistratiecomponent (Zaaktype, Statustypen, Resultaattypen,..)
+    * Domein/objectregistraties (domein/objectgegevens; meerdere registraties: één per domein/proces)
```

Comment:
Is dit een soort van rest categorie van objecten die je dan ordent per domein/proces?

Reply:
nee, geen rest categorie. Een erg belangrijke groep met specifieke gegevens per domein..
Dit is denk ik de grootste groep gegevens